### DISCUSSION TITLE
Module error running for first time

### DISCUSSION THREAD

**Discussion by fr3dd** [silverhack/monkey365#85] [General]
Greetings,

I have not yet used the module, but it seems like it is currently referencing a location that is not currently available. Since I have not run this module before, I am hesitant to simply remove line 19, (reference to /core/watcher), and try it again for fear that I am missing something obvious. Here is my error message:

Import-Module 'C:\Source\GitHub\monkey365'
Exception calling "EnumerateFiles" with "3" argument(s): "Could not find a part of the path
'C:\Source\GitHub\monkey365\core\watcher'."
At C:\Source\GitHub\monkey365\monkey365.psm1:35 char:5
+     [System.IO.Directory]::EnumerateFiles(("{0}{1}" -f $PSScriptRoot, ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : DirectoryNotFoundException

Please let me know if I can just remove the reference or if there is something I am missing.

**Comment by silverhack** [comment]
Hi @fr3dd, thanks for letting me know about that, it's really appreciated. 

I can confirm that it's a [bug](https://github.com/silverhack/monkey365/issues/87) and should be fixed shortly. 

The watcher module is a local routine that I'm using to watch some internal variables, but in the meantime, you can safely ignore that exception until a patch is available. I'll let you know once the develop branch is updated. 

Thanks! :D

**Comment by silverhack** [comment]
Hi @fr3dd, 

This is now fixed on dev branch. 

Cheers,
